experimental/dds/tree2/src/test/feature-libraries/defaultFieldKinds.spec.ts

```diff
@@ -174,22 +206,27 @@ describe("Value field changesets", () => {
 			change1WithChildChange,
 		);
 
-		const expected: FieldKindsTypes.ValueChangeset = {
-			value: { set: tree1 },
-			changes: nodeChange1,
+		const expected: FieldKindsTypes.OptionalChangeset = {
+			fieldChange: {
+				newContent: { set: tree1, changes: nodeChange1 },
+				id: brand(1),
+				wasEmpty: false,
+				revision: change1.revision,
+			},
 		};
 
 		assert.deepEqual(change1WithChildChange, expected);
```

Comment:
I don't know why we're bothering to recreate this data here. I think this assert and the associated `expected` variable can be removed.